README.md

```diff
@@ -22,6 +22,96 @@ The module has been tested with:
+   * Example: `n2-highmem-32` with 2 or more local SSDs
+
+### Enabling Disk Support
+
+To enable disk support with default settings in your Terraform configuration:
+
+```hcl
+enable_disk_support = true
+
+gke_config = {
+  node_count   = 3
+
+  # This machine has 256GB RAM
+  machine_type = "n2-highmem-32"
+
+  # This is for the OS disk, not for Materialize data
+  disk_size_gb = 100
+  min_nodes    = 3
+  max_nodes    = 5
+
+  # This provides 2 x 375GB = 750GB of local SSD storage
+  # Exceeding the 2:1 disk-to-RAM ratio (256GB RAM : 750GB disk)
+  local_ssd_count = 2
+}
+```
+
+This configuration:
+1. Attaches two local SSDs to each node, providing 750GB of storage per node
+2. Ensures the disk-to-RAM ratio is greater than 2:1 for the n2-highmem-32 instance (which has 256GB RAM)
+3. Installs OpenEBS via Helm to manage these local SSDs
+4. Configures local NVMe SSD devices using the [bootstrap](./modules/gke/bootstrap.sh) script
+5. Creates appropriate storage classes for Materialize
+
+### Advanced Configuration Example
+
+For a different machine type with appropriate disk sizing:
+
+```hcl
+enable_disk_support = true
+
+gke_config = {
+  node_count   = 3
+  machine_type = "n2-highmem-16"  # This machine has 128GB RAM
+  disk_size_gb = 100
+  min_nodes    = 3
+  max_nodes    = 5
+  local_ssd_count = 1             # This provides 1 x 375GB = 375GB of local NVMe SSD storage
+                                  # Exceeding the 2:1 disk-to-RAM ratio (128GB RAM : 375GB disk)
+}
+
+disk_support_config = {
+  openebs_version    = "4.2.0"
+  storage_class_name = "custom-storage-class"
+}
+```
+
+### Calculating the Right Number of Local SSDs
+
+The following table helps you determine the appropriate number of local SSDs based on your chosen machine type to maintain the recommended 2:1 disk-to-RAM ratio:
+
+| Machine Type    | RAM     | Required Disk | Recommended Local SSD Count | Total SSD Storage |
+|-----------------|---------|---------------|-----------------------------|-------------------|
+| `n2-highmem-8`  | `64GB`  | `128GB`       | 1                           | `375GB`           |
+| `n2-highmem-16` | `128GB` | `256GB`       | 1                           | `375GB`           |
+| `n2-highmem-32` | `256GB` | `512GB`       | 2                           | `750GB`           |
+| `n2-highmem-64` | `512GB` | `1024GB`      | 3                           | `1125GB`          |
+| `n2-highmem-80` | `640GB` | `1280GB`      | 4                           | `1500GB`          |
+
+Remember that each local NVMe SSD in GCP provides 375GB of storage.
+Choose the appropriate `local_ssd_count` to make sure your total disk space is at least twice the amount of RAM in your machine type for optimal Materialize performance.
+
+### Local SSD Limitations in GCP
+
+Note that there are some differences between AWS NVMe instance store and GCP local SSDs:
```

Comment:
Please remove this line. AWS is not relevant, and most of these "differences" are the same in AWS.